ClientApp/viking-vault/src/Common/styles.css

```diff
@@ -364,4 +364,21 @@
 
 .redirect-symbols:hover {
     text-decoration: none;
+}
+
+.white-container {
```

Comment:
It's a bit weird for a class to be named as white container in the first place. It's better to name it according to what the element that the class is attached to represents. What if for example you want to change the color theme of the website, then you'd also have to change the class name.

But in this case it's also weird because it doesn't contain any white color anywhere in the class properties